src/deploy-cromwell-on-azure/scripts/cromwell-application.conf

```diff
@@ -13,6 +13,19 @@ system {
   }
 }
 
+filesystems {
+    blob {
+        class = "cromwell.filesystems.blob.BlobPathBuilderFactory"
+        global {
+            class = "cromwell.filesystems.blob.BlobFileSystemManager"
+            config {
+                container: "cromwell-executions"
+                endpoint: "https://{StorageAccount}.blob.core.windows.net"
+            }
```

Comment:
This cromwell config works when the prerelease build 86-2a69691 but is now failing with the final 86 release.